Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml

```diff
@@ -0,0 +1,346 @@
+        - AdvMopItem
+        - RGBStaff
+        - WeaponAdvancedLaser
+        - AmmoTechFabCircuitboard
+        - PlushieArachind
+        - ClothingMaskArcadia
+        - ClothingBeltChiefEngineerFilled
+        - CowToolboxFilled
+        - ClothingShoesBootsMagCombatFilled
+        - ClothingShoesBootsMagGaloshes
+        - Hypospray
+        - Katana
+        - WeaponLaserCannon
+        - MuteToxinChemistryBottle
+        - ClothingOuterHardsuitCap
+        - ClothingShoesBootsMagBlinding
+        - TurboItemRechargerCircuitboard
+        - WeaponRevolverDeckard
+        - MedicatedSuture
+        - FoodPizzaArnold
+        - PinpointerUniversal
+        - Claymore
+      rareChance: 0.3
+
+- type: entity
+  id: RandomRarelegalLoot
+  name: Random Rare Legal Loot
+  suffix: Spawner 
+  parent: MarkerBase
+  components:
+    - type: Sprite
+      layers:
+        - state: red
+        - sprite: Objects/Misc/bedsheets.rsi
+          state: sheetUSA
+    - type: RandomSpawner
+      prototypes:
+        - ClothingBeltSheathFilled
+        - SpaceCash10000
+        - WeaponAntiqueLaser
+        - WeaponForceGun
+        - PenCentcom
+        - ClothingOuterArmorPodWars
+        - ClothingHeadHelmetPodWars
+        - ClothingOuterCoatSpaceAsshole
+        - UnholyHalberd
+        - ClockworkShield
+        - MirrorShield
+        - BedsheetUSA
+        - HypoMini
+        - HypoMiniLimitedEdition
+        - ClothingBackpackHolding
+        - ClothingShoesBootsMagAdv
+        - OreBagOfHolding
+        - ClothingShoesBootsSpeed
+        - ClothingBackpackDuffelHolding
+        - ClothingBackpackMessengerHolding
+        - MedkitCombatFilled
+        - TrashBagOfHolding
+        - AdvMopItem
+        - HandheldCrewMonitor
+        - WeaponAdvancedLaser
+        - VariedXenoArtifactItem
+        - ClothingMaskArcadia
+        - ClothingShoesBootsMagCombatFilled
+        - WeaponLaserCannon
+        - ClothingEyesHudMedSec
+        - PowerCellMicroreactor
+        - MuteToxinChemistryBottle
+        - SmokeGrenade
+        - ClothingShoesBootsMagBlinding
+        - TechnologyDiskRare
+        - Truncheon
+        - TurboItemRechargerCircuitboard
+        - ResearchDisk10000
+        - MedicatedSuture
+        - RegenerativeMesh
+        - FoodPizzaArnold
+        - PinpointerUniversal
+      chance: 1
+
+- type: entity
+  id: RandomEventIllegalLoot 
+  name: Random Illegal Loot
+  suffix: Spawner 
+  parent: MarkerBase
+  components:
+    - type: Sprite
+      layers:
+        - state: red
+        - sprite: Objects/Misc/bedsheets.rsi
+          state: sheetsyndie
+    - type: RandomSpawner
+      prototypes:
+      - DnaScramblerImplanter
+      - CyberPen
+      - CigPackSyndicate
+      - WeaponRevolverPythonAP
+      - ClothingEyesGlassesOutlawGlasses
+      - SyndicatePersonalAI
+      - EnergySword
+      - WeaponPistolCobra
+      - EmpGrenade
+      - Stimpack
+      - ClusterGrenade
+      - EncryptionKeyBinary
```

Comment:
Is this illegal?

Reply:
So I think it was post to be use to talk to borgs after you emag them to give them commands, Either away I remove it